src/ServicePulse.Host/vue/src/components/DataView.vue

```diff
@@ -0,0 +1,177 @@
+<script setup>
+import { computed, onMounted, onUnmounted, ref } from "vue";
+import { useFetchFromServiceControl } from "../composables/serviceServiceControlUrls";
+
+const props = defineProps({
+  apiUrl: String,
+  itemKey: {
+    type: String,
+    default: "id",
+  },
+  itemsPerPageOptions: {
+    type: Array,
+    default: () => [],
```

Comment:
I feel this could be defaulted. 
I don't really see a reason for all consumers to have to think about it, thoughts?

Reply:
I did not default it because I wanted the default to be not to show it at all. I defaulted the values and included a boolean flag to show or hide it so it works the same as the pagination.